.github/workflows/python-publish.yml

```diff
@@ -20,8 +20,8 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel numpy
-    - name: Build the package
-      run: python setup.py sdist
+    #- name: Build the package
+    #  run: python setup.py sdist
```

Comment:
@odstrcilt merging something like this needs to be done carefully... what's your reasoning here? Are we giving up on distributing via PyPi until we can manage the omfit_classes dependence?

Reply:
Because the setup.py file is no longer there, Enrico has removed it. The regression tests passed, it it has somehow installed, so I expect that it works.